diagnostic.md

```diff
@@ -35,7 +42,13 @@ Record your responses inside the fenced code blocks below each question.
     ```
 
     ```md
-    <!-- your response here -->
+    The first route is saying that it is a nested route and that product model will be listed under the products model.
+    What happens with the first route is that the products model
+    will first render and then the product model will render inside
+    of it where there is an outlet.
+
+    The latter is not nested but still a dynamic route, product model
+    will load independently of products model. 
```

Comment:
Looks Good!  These two route definitions are identical URL paths.  Nested route definitions, do require a nested file structure, and the big distinction is where Ember will render the content.  
The nested `'products'/'product'` instructs Ember to render
`app/products/product/template.hbs` in the `outlet` of the parent `app/products/template.hbs`.  
The non-nested `'product'` instructs Ember to render `app/product/template.hbs` in the context of the application template.